flump/exceptions.py

```diff
@@ -2,7 +2,7 @@
 
 
 class FlumpUnprocessableEntity(UnprocessableEntity):
-    def __init__(self, *args, errors=None, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, errors=None, *args, **kwargs):
+        super(UnprocessableEntity, self).__init__(*args, **kwargs)
```

Comment:
super's argument should be `FlumpUnprocessableEntity` (unless you're intentionally jumping to the great-parent, but's not usually the case)